website/src/pages/Blog/index.tsx

```diff
@@ -292,15 +270,11 @@ const Blog: React.FC = () => {
         <>
           <div className={classes.root}>
             <Container maxWidth="md">
-              <p
-                className={classes.authorURL}
-              >{`/blog/${queryTagName}/${queryTitleName}/${queryAuthorName}`}</p>
+              <p className={classes.authorURL}>{`/blog/${queryAuthorName}`}</p>
               <div className={classes.authorWrapper}>
                 <Avatar
-                  alt={queryAuthorName ? queryAuthorName : ""}
-                  src={`../Images/blog/authors/${queryAuthorName.toLowerCase()
-                    .replace(/[^\w ]+/g,'')
-                    .replace(/ +/g,'-')}.png`}
+                  alt={queryAuthorName && queryAuthorName}
+                  src={`../Images/blog/authors/${getAvatar(queryAuthorName)}.png`}
```

Comment:
we have a util right? for getting the avatar

Reply:
oh okay.. missed the comments

Reply:
yes, it returns the authorName after some replaced character, so to make it generic instead of sending the entire hardcoded path we are returning only the name.